code/modules/shuttle/mini_dropship.dm

```diff
@@ -10,33 +10,184 @@
 	dheight = 0
 	width = 7
 	height = 9
-	callTime = 15 SECONDS
-	rechargeTime = 3 MINUTES
 
 /obj/machinery/computer/camera_advanced/shuttle_docker/minidropship
 	name = "Tadpole navigation computer"
 	desc = "Used to designate a precise transit location for the Tadpole."
-	icon_state = "maptable"
-	resistance_flags = UNACIDABLE|INDESTRUCTIBLE
+	icon_state = "shuttlecomputer"
 	req_one_access = list(ACCESS_MARINE_DROPSHIP, ACCESS_MARINE_LEADER)
+	density = FALSE
```

Comment:
Why

Reply:
Xenos can melt the windows, so unless the density is false, or we must allow them to kill it